packages/btcindexer/wrangler.jsonc

```diff
@@ -94,4 +94,26 @@
       "id": "93968151e21e4cfa8f23041c0a5baf08",
     },
   ],
+  "queues": {
+    "consumers": [
+      {
+        "queue": "regtest-block-queue",
+        "max_batch_size": 10,
+        "max_batch_timeout": 30,
+        "max_retries": 3,
+      },
+      {
+        "queue": "testnet-block-queue",
+        "max_batch_size": 10,
+        "max_batch_timeout": 30,
+        "max_retries": 3,
+      },
+      {
+        "queue": "mainnet-block-queue",
+        "max_batch_size": 1,
+        "max_batch_timeout": 5,
+        "max_retries": 3,
+      },
+    ],
+  },
```

Comment:
if we have only one worker for handling all networks, then I think we should have only one queue as well.

Reply:
changed it to one queue